securemail/securemail.php

```diff
@@ -81,7 +81,7 @@ function securemail_settings(App &$a, &$s){
  *
  * @see App
  */
-function securemail_settings_post(App &$a, array &$b){
+function securemail_settings_post(App $a, array &$b){
```

Comment:
Please add a new line before opening brace. I'm asking because it's also missing a space before the brace, might as well going for the right thing.